### DISCUSSION TITLE
Weird looking stop status in notifications panel

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15653] [Q&A] [answered]
As its ovelaps previous status it looks kinda weird and hard to read:
![ScreenShot_20211116182424](https://user-images.githubusercontent.com/86934170/142015949-c95aa714-8dd8-4d8d-b175-5fba01a1428e.png)
So maybe it should be red painted/adjusted with a size?
![ScreenShot_20211116182424333](https://user-images.githubusercontent.com/86934170/142015970-067a8e66-f929-4774-a3a4-be99d855230e.png)

**Comment by peppy** [comment, marked as answer]
Probably shouldn't have a cancel button. Update doesn't support cancel.

**Reply by Loreos7** [reply]
What if installation has failed? As for https://github.com/ppy/osu/pull/15655 this icon should be changed.
![ScreenShot_20211117185308](https://user-images.githubusercontent.com/86934170/142235729-6a929163-612f-4995-8e12-d3ea036b11af.png)

**Reply by peppy** [reply]
Is a good point. That specific case still needs to be handled better.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/15783
